rest/api.go

```diff
@@ -290,6 +303,7 @@ func (h *handler) handlePostResync() error {
 			err := h.db.ResyncManager.Start(h.ctx(), map[string]interface{}{
 				"database":            h.db,
 				"regenerateSequences": regenerateSequences,
+				"collections":         resyncPostReqBody,
```

Comment:
I think I wasn't clear in my previous review, but I meant that 'collections' should be getting passed down to ResyncManagerStart as just a set of collections, as opposed to the REST payload.  i.e. the ResyncManager.Start should define the options used, and should be independent of this particular usage from the REST API.  So define ResyncPostReqBody here in the REST package, and define the struct for collections only in db.  It's fine if that ends up looking like:
```
in db:
type ResyncCollections map[string][]string

in rest:
type ResyncPostReqBody struct {
	Scope               db.ResyncCollections `json:"scopes,omitempty"`
	RegenerateSequences bool                `json:"regenerate_sequences,omitempty"`
}
```
The main point is just to avoid defining the REST payload down in the db package, to better support other use cases in future (as well as avoid the redundant definition of regenerateSequences).

Reply:
oh I see. Fixed it